app/templates/components/mediasuite-radio-button.hbs

```diff
@@ -1,3 +1,11 @@
-<input type="radio" name={{name}} value={{value}} checked={{isChecked}} disabled={{isDisabled}} id={{inputID}}>
+{{one-way-radio
+  name=name
+  value=value
+  option=option
+  disabled=isDisabled
+  id=inputID
+  update=update
+  class=class
+}}
```

Comment:
I'm assuming you checked for the whitespace click bug we had previously with the radio button input, yes?